docs/gettingStarted/setupGenericScheme.md

```diff
@@ -0,0 +1,130 @@
+# Setup Generic Scheme for a DAO
+
+## When to use Generic Scheme
+[Generic Scheme](https://github.com/daostack/arc/blob/master/contracts/schemes/GenericScheme.sol) allows a DAO to interact with other contracts and execute any method from the contract if the DAO decides to
+
+## How to register Generic Scheme with a DAO
+
+DAO only uses the schemes that are registered with its controller. There are two ways to register a scheme:
+
+  - While forging/deploying the DAO as part of initial set of schemes
+
+  - While proposing the new scheme to DAO via Scheme Registrar for the DAO.
```

Comment:
in general each scheme which has a permission to register scheme can do that .https://github.com/daostack/arc/blob/master/contracts/controller/Controller.sol#L21
Registrar scheme has this permission